From 5fb169023c98becf99bc960a95134f230fcdda62 Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Date: Wed, 15 May 2019 16:03:59 +0000
Subject: [PATCH] sql: Include <utility> for std::pair

IWYU after commit 153dd1a64487 ("sqlite: Rewrite custom recovery code"),
which also fixes the libstdc++ build:

../../sql/recover_module/integers.h:67:6: error: no template named 'pair' in namespace 'std'
std::pair<int64_t, const uint8_t*> ParseVarint(const uint8_t* buffer,
~~~~~^

Bug: 957519
Change-Id: Iee9ae4cfb1c3531b6fcd2080f5b9a3bcfc25f5cc
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/1611690
Auto-Submit: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Commit-Queue: Victor Costan <pwnall@chromium.org>
Reviewed-by: Victor Costan <pwnall@chromium.org>
Cr-Commit-Position: refs/heads/master@{#659956}
---

diff --git a/sql/recover_module/integers.h b/sql/recover_module/integers.h
index 8b19297..f02d0c2 100644
--- a/sql/recover_module/integers.h
+++ b/sql/recover_module/integers.h
@@ -7,6 +7,7 @@
 
 #include <cstdint>
 #include <limits>
+#include <utility>
 
 namespace sql {
 namespace recover {
